### ISSUE TITLE
Release 2021.07 - RC1

### ISSUE THREAD

**Issue by MrKevinWeiss** [RIOT-OS/Release-Specs#227]
This issue lists the status of all tests for the [Release Candidate 1 of the 2021.07](https://github.com/RIOT-OS/RIOT/releases/tag/2021.07-RC1) release.

Specs tested:
- [ ] [01-ci](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/01-ci/README.md)
  - [ ] [Task #01 - Compile test](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/01-ci/README.md#task-01---compile-test)
  - [x] [Task #02 - Unittests on native](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/01-ci/README.md#task-02---unittests-on-native) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874039214) 
  - [x] [Task #03 - Unittests on native separated](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/01-ci/README.md#task-03---unittests-on-native-separated) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874039336) 
  - [x] [Task #04 - Unittests on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/01-ci/README.md#task-04---unittests-on-iotlab-m3) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874039893) 
- [ ] [02-tests](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/02-tests/README.md)
  - [ ] [Task #01 - All tests on native](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/02-tests/README.md#task-01---all-tests-on-native)
  - [ ] [Task #02 - Subset of tests on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/02-tests/README.md#task-02---subset-of-tests-on-iotlab-m3)
  - [ ] [Task #03 - Subset of tests on samr21-xpro](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/02-tests/README.md#task-03---subset-of-tests-on-samr21-xpro)
- [x] [03-single-hop-ipv6-icmp](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/03-single-hop-ipv6-icmp/README.md)
  - [x] [Task #01 - ICMPv6 multicast echo on native](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/03-single-hop-ipv6-icmp/README.md#task-01---icmpv6-multicast-echo-on-native) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874044332) 
  - [x] [Task #02 - ICMPv6 link-local echo on native](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/03-single-hop-ipv6-icmp/README.md#task-02---icmpv6-link-local-echo-on-native) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874045404) 
  - [x] [Task #03 - ICMPv6 link-local echo on native (1 hour)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/03-single-hop-ipv6-icmp/README.md#task-03---icmpv6-link-local-echo-on-native-1-hour) @riot-ci 
  - [x] [Task #04 - ICMPv6 stress test on native (1 hour)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/03-single-hop-ipv6-icmp/README.md#task-04---icmpv6-stress-test-on-native-1-hour) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874122531) 
  - [x] [Task #05 - ICMPv6 stress test on native (neighbor cache stress)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/03-single-hop-ipv6-icmp/README.md#task-05---icmpv6-stress-test-on-native-neighbor-cache-stress) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874123469) 
  - [x] [Task #06 - ICMPv6 link-local echo on native (IPv6 fragmentation)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/03-single-hop-ipv6-icmp/README.md#task-06---icmpv6-link-local-echo-on-native-ipv6-fragmentation) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874124846) 
- [ ] [04-single-hop-6lowpan-icmp](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md)
  - [x] [Task #01 - ICMPv6 link-local echo with iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-01---icmpv6-link-local-echo-with-iotlab-m3) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874040873) 
  - [x] [Task #02 - ICMPv6 multicast echo with iotlab-m3/samr21-xpro](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-02---icmpv6-multicast-echo-with-iotlab-m3samr21-xpro) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874042861) 
  - [x] [Task #03 - ICMPv6 echo with large payload](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-03---icmpv6-echo-with-large-payload) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874044777) 
  - [x] [Task #04 - ICMPv6 echo with iotlab-m3/samr21-xpro 15 minutes](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-04---icmpv6-echo-with-iotlab-m3samr21-xpro-15-minutes) @riot-ci 
  - [ ] [Task #05 (Experimental) - ICMPv6 multicast echo with samr21-xpro/cc2538](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-05-experimental---icmpv6-multicast-echo-with-samr21-xprocc2538)
  - [ ] [Task #06 (Experimental)- ICMPv6 link-local echo with samr21-xpro/cc2538](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-06-experimental---icmpv6-link-local-echo-with-samr21-xprocc2538)
  - [ ] [Task #07 (Experimental) - ICMPv6 multicast echo with samr21-xpro/zero + xbee](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-07-experimental---icmpv6-multicast-echo-with-samr21-xprozero--xbee)
  - [ ] [Task #08 (Experimental) - ICMPv6 echo with samr21-xpro/zero + xbee](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-08-experimental---icmpv6-echo-with-samr21-xprozero--xbee)
  - [x] [Task #09 - ICMPv6 stress test on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-09---icmpv6-stress-test-on-iotlab-m3) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874075407) 
  - [x] [Task #10 (Experimental) - ICMPv6 echo with large payload (IPv6 fragmentation)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-10-experimental---icmpv6-echo-with-large-payload-ipv6-fragmentation) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874077264) 
  - [x] [Task #11 (Experimental) - ICMPv6 stress test on nrf802154](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-11-experimental---icmpv6-stress-test-on-nrf802154) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874078372) 
  - [x] [Task #12 (Experimental) - ICMPv6 multicast echo with iotlab-m3/nrf802154](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-12-experimental---icmpv6-multicast-echo-with-iotlab-m3nrf802154) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874080008) 
  - [x] [Task #13 (Experimental) - ICMPv6 link-local echo with iotlab-m3/nrf802154](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/04-single-hop-6lowpan-icmp/README.md#task-13-experimental---icmpv6-link-local-echo-with-iotlab-m3nrf802154) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874081543) 
- [x] [05-single-hop-route](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/05-single-hop-route/README.md)
  - [x] [Task #01 - ICMPv6 echo unicast addresess on native (default route)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/05-single-hop-route/README.md#task-01---icmpv6-echo-unicast-addresess-on-native-default-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874125169) 
  - [x] [Task #02 - ICMPv6 echo unicast addresess on iotlab-m3 (default route)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/05-single-hop-route/README.md#task-02---icmpv6-echo-unicast-addresess-on-iotlab-m3-default-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874082454) 
  - [x] [Task #03 - ICMPv6 echo unicast addresess on native (specific route)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/05-single-hop-route/README.md#task-03---icmpv6-echo-unicast-addresess-on-native-specific-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874125442) 
  - [x] [Task #04 - ICMPv6 echo unicast addresess on iotlab-m3 (static route)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/05-single-hop-route/README.md#task-04---icmpv6-echo-unicast-addresess-on-iotlab-m3-static-route) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874125703) 
- [x] [06-single-hop-udp](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/06-single-hop-udp/README.md)
  - [x] [Task #01 - UDP on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/06-single-hop-udp/README.md#task-01---udp-on-iotlab-m3) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874107571) 
  - [x] [Task #02 - UDP on iotlab-m3 (UDP port compression)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/06-single-hop-udp/README.md#task-02---udp-on-iotlab-m3-udp-port-compression) @miri64 (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874717209) 
  - [x] [Task #03 - UDP on native (non-existent neighbor)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/06-single-hop-udp/README.md#task-03---udp-on-native-non-existent-neighbor) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874125873) 
  - [x] [Task #04 - UDP on iotlab-m3 (non-existent neighbor)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/06-single-hop-udp/README.md#task-04---udp-on-iotlab-m3-non-existent-neighbor) @miri64 (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874717209) 
  - [x] [Task #05 - Empty UDP on native](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/06-single-hop-udp/README.md#task-05---empty-udp-on-native) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874126365) 
  - [x] [Task #06 - Empty UDP on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/06-single-hop-udp/README.md#task-06---empty-udp-on-iotlab-m3) @miri64 (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874717209) 
- [ ] [07-multi-hop](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/07-multi-hop/README.md)
  - [x] [Task #01 - ICMPv6 echo on iotlab-m3 with three hops (static route)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/07-multi-hop/README.md#task-01---icmpv6-echo-on-iotlab-m3-with-three-hops-static-route) @miri64 (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874717209) 
  - [x] [Task #02 - UDP on iotlab-m3 with three hops (static route)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/07-multi-hop/README.md#task-02---udp-on-iotlab-m3-with-three-hops-static-route) @miri64 (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874717209) 
  - [x] [Task #03 - ICMPv6 echo on iotlab-m3 with three hops (RPL route)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/07-multi-hop/README.md#task-03---icmpv6-echo-on-iotlab-m3-with-three-hops-rpl-route) @miri64 (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874717209) 
  - [x] [Task #04 - UDP on iotlab-m3 with three hops (RPL route)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/07-multi-hop/README.md#task-04---udp-on-iotlab-m3-with-three-hops-rpl-route) @miri64 (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874717209) 
  - [ ] [Task #05 (Experimental) - UDP with large payload on iotlab-m3 with three hops (RPL route)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/07-multi-hop/README.md#task-05-experimental---udp-with-large-payload-on-iotlab-m3-with-three-hops-rpl-route)
- [ ] [08-interop](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md)
  - [x] [Task #01 - ICMPv6 echo between native and Linux](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md#task-01---icmpv6-echo-between-native-and-linux) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874126649) 
  - [ ] [Task #02 - ICMPv6 echo between iotlab-m3 and Linux with 6LowPAN](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md#task-02---icmpv6-echo-between-iotlab-m3-and-linux-with-6lowpan)
  - [ ] [Task #03 - UDP exchange between iotlab-m3 and Contiki](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md#task-03---udp-exchange-between-iotlab-m3-and-contiki)
  - [ ] [Task #04 - ICMPv6 echo between iotlab-m3 and Internet host through Linux with 6LowPAN](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md#task-04---icmpv6-echo-between-iotlab-m3-and-internet-host-through-linux-with-6lowpan)
  - [ ] [Task #05 - ICMPv6 echo between iotlab-m3 and Internet host through RIOT border router](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md#task-05---icmpv6-echo-between-iotlab-m3-and-internet-host-through-riot-border-router)
  - [ ] [Task #06 - UDP between iotlab-m3 and Internet host through RIOT border router](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md#task-06---udp-between-iotlab-m3-and-internet-host-through-riot-border-router)
  - [ ] [Task #07 - UDP between iotlab-m3 and Internet host through RIOT border router (200b payload)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md#task-07---udp-between-iotlab-m3-and-internet-host-through-riot-border-router-200b-payload)
  - [x] [Task #08 - UDP between GNRC and lwIP on iotlab-m3](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md#task-08---udp-between-gnrc-and-lwip-on-iotlab-m3) @miri64 (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874717209) 
  - [x] ~~Task #09 - deprecated~~
  - [x] ~~Task #10 - deprecated~~
  - [ ] [Task #11 - UDP exchange between iotlab-m3 and Zephyr](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md#task-11---udp-exchange-between-iotlab-m3-and-zephyr)
  - [ ] [Task #12 - GNRC Border Router with WiFi uplink to the Internet](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md#task-12---gnrc-border-router-with-wifi-uplink-to-the-internet)
  - [ ] [Task #13 - GNRC Networking](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/08-interop/README.md#task-13---gnrc-networking)
- [x] [09-coap](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/09-coap/README.md)
  - [x] [Task #01 - CORD Endpoint](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/09-coap/README.md#task-01---cord-endpoint) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874127472) 
  - [x] [Task #02 - Confirmable retries](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/09-coap/README.md#task-02---confirmable-retries) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874127729) 
  - [x] [Task #03 - Block1](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/09-coap/README.md#task-03---block1) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874130250) 
  - [x] [Task #04 - Block2](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/09-coap/README.md#task-04---block2) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874131644) 
  - [x] [Task #05 - Observe registration and notification](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/09-coap/README.md#task-05---observe-registration-and-notification) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874131774) 
- [x] [10-icmpv6-error](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/10-icmpv6-error/README.md)
  - [x] [Task #01 - Destination unreachable - no route to destination](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/10-icmpv6-error/README.md#task-01---destination-unreachable---no-route-to-destination) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874131859) 
  - [x] [Task #02 - Destination unreachable - Beyond scope of source address](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/10-icmpv6-error/README.md#task-02---destination-unreachable---beyond-scope-of-source-address) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874131953) 
  - [x] [Task #03 - Destination unreachable - address unreachable (target node address)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/10-icmpv6-error/README.md#task-03---destination-unreachable---address-unreachable-target-node-address) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874132050) 
  - [x] [Task #04 - Destination unreachable - address unreachable (neighbor cache miss)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/10-icmpv6-error/README.md#task-04---destination-unreachable---address-unreachable-neighbor-cache-miss) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874132203) 
  - [x] [Task #05 - Destination unreachable - port unreachable](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/10-icmpv6-error/README.md#task-05---destination-unreachable---port-unreachable) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874132309) 
  - [x] [Task #06 - Destination unreachable - port unreachable (large payload)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/10-icmpv6-error/README.md#task-06---destination-unreachable---port-unreachable-large-payload) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874132410) 
  - [x] [Task #07 - Packet too big](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/10-icmpv6-error/README.md#task-07---packet-too-big) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874132639) 
  - [x] [Task #08 - Time exceeded - hop limit exceeded in transit](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/10-icmpv6-error/README.md#task-08---time-exceeded---hop-limit-exceeded-in-transit) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874132811) 
  - [x] [Task #09 - Parameter problem - erroneous header field encountered](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/10-icmpv6-error/README.md#task-09---parameter-problem---erroneous-header-field-encountered) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874132905) 
  - [x] [Task #10 - IPv6-in-IPv6 encapsulation](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/10-icmpv6-error/README.md#task-10---ipv6-in-ipv6-encapsulation) @riot-ci (see https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874133027) 
- [x] [11-lorawan](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md)
  - [x] [Task #01 - LoRaWAN example](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-01---lorawan-example) @jia200x (PASSED: https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874677092)
  - [x] [Task #02 - OTAA join procedure](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-02---otaa-join-procedure) @jia200x (PASSED: https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-875709309)
  - [x] [Task #03 - ABP join procedure](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-03---abp-join-procedure) @jia200x (PASSED: https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-875717492)
  - [x] [Task #04 - LoRaWAN device parameters persistence](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-04---lorawan-device-parameters-persistence) @jia200x (PASSED: https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-875722545)
  - [x] [Task #05 - GNRC LoRaWAN (Over The Air Activation)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-05---gnrc-lorawan-over-the-air-activation) @jia200x (PASSED: https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-875728274)
  - [x] [Task #06 - GNRC LoRaWAN (ABP)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-06---gnrc-lorawan-abp) @jia200x (PASSED: https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-875732610)
- [ ] [99-compile-and-test-one-board](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/99-compile-and-test-one-board/README.md)
  - [ ] [Task #01 - Run tests on different hardwares](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/99-compile-and-test-one-board/README.md#task-01---run-tests-on-different-hardwares)

**Comment by MrKevinWeiss** [comment]
Expect that the lorawan tests will fail as updates to the TTN seem to break things with the IoT lab lora tests.  @jia200x has kindly volunteered to run some manual tests until the issues are resolved.

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
https://github.com/RIOT-OS/Release-Specs/issues/227#issuecomment-874068904 => just testing something. Ignore for now.

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
> # Test Report

See https://github.com/RIOT-OS/Release-Specs/pull/228

**Comment by miri64** [comment]
> heavy_multiplication_x	[01. Task 02 - Unittests on native](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/01-ci/README.md#task-02---unittests-on-native)	
> **FAILED**

See https://github.com/RIOT-OS/RIOT/pull/16614.

**Comment by miri64** [comment]
> yellow_circle	[03. Task 06 - ICMPv6 link-local echo on native (IPv6 fragmentation)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/03-single-hop-ipv6-icmp/README.md#task-06---icmpv6-link-local-echo-on-native-ipv6-fragmentation)	**SKIPPED** (Skipped: tap0 and tap1 not bridged)

Did that intentionally to have an example for `SKIPPED` ;-).

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by jia200x** [comment]
### [11. Task #01 - LoRaWAN example](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-01---lorawan-example)

<details><summary><strong>PASSED</strong></summary>

```
2021-07-06 12:14:48,863 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-06 12:14:48,865 # LoRaWAN Class A low-power application
2021-07-06 12:14:48,867 # =====================================
2021-07-06 12:14:48,868 # Starting join procedure
2021-07-06 12:14:53,196 # Join procedure succeeded
2021-07-06 12:14:53,198 # Sending: This is RIOT!
2021-07-06 12:15:14,310 # Sending: This is RIOT!
2021-07-06 12:15:23,032 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
```
</details>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by MrKevinWeiss** [comment]
So it looks like there are still some missing experimental tests (something I can live with) for 04-single-hop-6lowpan-icmp and 07-multihop, some interop tests (any volunteers?) and some some lorawan tests (though I think @jia200x is working on them).  Are there any more bugs that will need to be backported before RC-2.  I think I would wait until the nice formatting that @miri64 has been working on is in before starting it.

I believe the unit tests errors have been corrected with https://github.com/RIOT-OS/RIOT/pull/16614 thanks to @kaspar030 and @maribu

**Comment by miri64** [comment]
> I believe the unit tests errors have been corrected with [RIOT-OS/RIOT#16614](https://github.com/RIOT-OS/RIOT/pull/16614) thanks to @kaspar030 and @maribu

Yepp. And they do not appear with Murdock, which has the more canonical results (build on Ubuntu vs. build on Arch with my machine).

**Comment by miri64** [comment]
<h1>Test Report</h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by jia200x** [comment]
### [11. Task #02 - OTAA join procedure](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-02---otaa-join-procedure)

<details><summary><strong>PASSED</strong></summary>

```
> loramac tx test uncnf 1232021-07-07 17:28:09,205 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 17:28:09,208 # All up, running the shell now
set appkset deveui 488038d49a5a156b
2021-07-07 17:28:12,184 # loramac set deveui 488038d49a5a156b
> loramaset appkey 8849748cafbc089b815f70b20b08af02
2021-07-07 17:28:14,121 # loramac set appkey 8849748cafbc089b815f70b20b08af02
> loramajoin otaa
2021-07-07 17:28:18,439 # loramac join otaa
2021-07-07 17:28:21,030 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 17:28:21,033 # All up, running the shell now
> loramaset deveui 488038d49a5a156b
2021-07-07 17:28:22,861 # loramac set deveui 488038d49a5a156b
> loramaset appkey 8849748cafbc089b815f70b20b08af02
2021-07-07 17:28:23,712 # loramac set appkey 8849748cafbc089b815f70b20b08af02
> loramaset dr 0
2021-07-07 17:28:28,722 # loramac set dr 0
loramac join otaa
2021-07-07 17:29:22,833 # loramac join otaa
2021-07-07 17:29:31,204 # Join procedure succeeded!
loramac tx test cnf 123
2021-07-07 17:30:40,764 # loramac tx test cnf 123
2021-07-07 17:30:44,301 # Received ACK from network
2021-07-07 17:30:44,303 # Message sent with success
> loramac tx test uncnf 42
2021-07-07 17:33:23,598 # loramac tx test uncnf 42
2021-07-07 17:33:27,630 # Message sent with success
> reboot
2021-07-07 17:33:33,610 # reboot
2021-07-07 17:33:33,697 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 17:33:33,700 # All up, running the shell now
loramac set deveui 488038d49a5a156b
2021-07-07 17:33:36,267 # loramac set deveui 488038d49a5a156b
loramac set appkey 8849748cafbc089b815f70b20b08af02
2021-07-07 17:33:37,772 # loramac set appkey 8849748cafbc089b815f70b20b08af02
loramac set dr 3
2021-07-07 17:33:41,526 # loramac set dr 3
loramac join otaa
2021-07-07 17:33:44,150 # loramac join otaa
2021-07-07 17:33:49,658 # Join procedure succeeded!
loramac tx test cnf 123
2021-07-07 17:33:52,729 # loramac tx test cnf 123
2021-07-07 17:33:54,078 # Received ACK from network
2021-07-07 17:33:54,080 # Message sent with success
loramac tx test uncnf 42
2021-07-07 17:34:48,878 # loramac tx test uncnf 42
2021-07-07 17:34:51,909 # Message sent with success
> reboot
2021-07-07 17:34:54,640 # reboot
2021-07-07 17:34:54,727 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 17:34:54,729 # All up, running the shell now
loramac set deveui 488038d49a5a156b
2021-07-07 17:34:58,532 # loramac set deveui 488038d49a5a156b
loramac set appkey 8849748cafbc089b815f70b20b08af02
2021-07-07 17:35:00,836 # loramac set appkey 8849748cafbc089b815f70b20b08af02
loramac set dr 3
2021-07-07 17:35:02,555 # loramac set dr 3
> loramac set dr 5
2021-07-07 17:35:04,795 # loramac set dr 5
loramac join otaa
2021-07-07 17:35:07,549 # loramac join otaa
2021-07-07 17:35:12,734 # Join procedure succeeded!
loramac tx test cnf 123
2021-07-07 17:35:16,764 # loramac tx test cnf 123
2021-07-07 17:35:17,893 # Received ACK from network
2021-07-07 17:35:17,895 # Message sent with success
loramac tx test uncnf 42
2021-07-07 17:35:44,850 # loramac tx test uncnf 42
2021-07-07 17:35:47,883 # Message sent with success
> 
```

</details>

**Comment by jia200x** [comment]
### [11. Task #03 - ABP join procedure](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-03---abp-join-procedure)

<details><summary><strong>PASSED</strong></summary>

```
2021-07-07 17:38:01,056 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 17:38:01,059 # All up, running the shell now
> loramac set devaddr 008f963a
2021-07-07 17:38:43,726 # loramac set devaddr 008f963a
> loramac set nwkske3cd35f918e7f1232ee0941778ee0238a8a
2021-07-07 17:39:33,085 # loramac set nwkskey 3cd35f918e7f1232ee0941778ee0238a
> loramac set appske5c140708b42dd430df7adfed1b401cefef
2021-07-07 17:39:43,047 # loramac set appskey 5c140708b42dd430df7adfed1b401cef
> loramac join abp
2021-07-07 17:39:45,780 # loramac join abp
2021-07-07 17:39:45,783 # Join procedure succeeded!
> loramac tx test cnf 123
2021-07-07 17:41:00,419 # loramac tx test cnf 123
2021-07-07 17:41:04,779 # Received ACK from network
2021-07-07 17:41:04,782 # Message sent with success
> loramac tx test uncnf 42
2021-07-07 17:41:40,762 # loramac tx test uncnf 42
2021-07-07 17:42:14,077 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 17:42:14,079 # All up, running the shell now
> loramaset nwkskey 3cd35f918e7f1232ee0941778ee0238a
2021-07-07 17:42:16,292 # loramac set nwkskey 3cd35f918e7f1232ee0941778ee0238a
> loramaset appskey 5c140708b42dd430df7adfed1b401cef
2021-07-07 17:42:18,088 # loramac set appskey 5c140708b42dd430df7adfed1b401cef
> loramaset devaddr 008f963a
2021-07-07 17:42:21,882 # loramac set devaddr 008f963a
loramac set dr 0
2021-07-07 17:42:29,873 # loramac set dr 0
> loramajoin abp
2021-07-07 17:42:33,968 # loramac join abp
2021-07-07 17:42:33,971 # Join procedure succeeded!
> loramatx test cnf 123
2021-07-07 17:42:37,371 # loramac tx test cnf 123
2021-07-07 17:42:41,236 # Received ACK from network
2021-07-07 17:42:41,238 # Message sent with success
loramac tx test uncnf 42
2021-07-07 17:44:23,765 # loramac tx test uncnf 42
2021-07-07 17:44:27,799 # Message sent with success
> 2021-07-07 17:44:33,773 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 17:44:33,776 # All up, running the shell now
> loramaset devaddr 008f963a
2021-07-07 17:44:38,810 # loramac set devaddr 008f963a
> loramaset nwkskey 3cd35f918e7f1232ee0941778ee0238a
2021-07-07 17:44:41,119 # loramac set nwkskey 3cd35f918e7f1232ee0941778ee0238a
> loramaset appskey 5c140708b42dd430df7adfed1b401cef
2021-07-07 17:44:43,918 # loramac set appskey 5c140708b42dd430df7adfed1b401cef
> loramaset dr 3
2021-07-07 17:44:50,784 # loramac set dr 3
> loramajoin abpef
2021-07-07 17:44:54,817 # loramac join abp
2021-07-07 17:44:54,819 # Join procedure succeeded!
> loramatx test cnf 123ef
2021-07-07 17:44:59,032 # loramac tx test cnf 123
2021-07-07 17:45:00,379 # Received ACK from network
2021-07-07 17:45:00,381 # Message sent with success
> loramatx test uncnf 42
2021-07-07 17:45:50,881 # loramac tx test uncnf 42
2021-07-07 17:45:53,913 # Message sent with success
> reboot
2021-07-07 17:45:56,513 # reboot
2021-07-07 17:45:56,599 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 17:45:56,602 # All up, running the shell now
loramac set devaddr 008f963a
2021-07-07 17:45:59,233 # loramac set devaddr 008f963a
loramac set nwkskey 3cd35f918e7f1232ee0941778ee0238a
2021-07-07 17:46:01,158 # loramac set nwkskey 3cd35f918e7f1232ee0941778ee0238a
loramac set appskey 5c140708b42dd430df7adfed1b401cef
2021-07-07 17:46:04,875 # loramac set appskey 5c140708b42dd430df7adfed1b401cef
loramac set dr 5
2021-07-07 17:46:07,640 # loramac set dr 5
loramac join abp
2021-07-07 17:46:09,772 # loramac join abp
2021-07-07 17:46:09,775 # Join procedure succeeded!
loramac tx test cnf 123
2021-07-07 17:46:11,814 # loramac tx test cnf 123
2021-07-07 17:46:12,943 # Received ACK from network
2021-07-07 17:46:12,946 # Message sent with success
loramac tx test uncnf 42
2021-07-07 17:46:21,009 # loramac tx test uncnf 42
2021-07-07 17:46:24,042 # Message sent with success
> 
```

</details>

**Comment by jia200x** [comment]
### [11. Task #04 - LoRaWAN device parameters persistence](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-04---lorawan-device-parameters-persistence)

<details><summary><strong>PASSED</strong></summary>

```
2021-07-07 17:51:24,604 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 17:51:24,607 # All up, running the shell now
> loramac get deveui
2021-07-07 17:51:31,567 # loramac get deveui
2021-07-07 17:51:31,569 # DEVEUI: 0000000000000000
> loramac get appeui
2021-07-07 17:51:38,718 # loramac get appeui
2021-07-07 17:51:38,721 # APPEUI: 0000000000000000
> loramac get appkey
2021-07-07 17:51:41,577 # loramac get appkey
2021-07-07 17:51:41,581 # APPKEY: 00000000000000000000000000000000
> loramac get devaddr
2021-07-07 17:51:44,045 # loramac get devaddr
2021-07-07 17:51:44,047 # DEVADDR: 00000000
> loramac get appskey
2021-07-07 17:51:47,580 # loramac get appskey
2021-07-07 17:51:47,584 # APPSKEY: 00000000000000000000000000000000
> loramac get nwkskey
2021-07-07 17:51:50,796 # loramac get nwkskey
2021-07-07 17:51:50,800 # NWKSKEY: 00000000000000000000000000000000
loramac set deveui 001122334455667777
2021-07-07 17:51:57,097 # loramac set deveui 0011223344556677
loramac set appeui 001122334455667777
2021-07-07 17:52:01,444 # loramac set appeui 0011223344556677
loramac set appkey 0011223344556677001122334455667777
2021-07-07 17:52:04,933 # loramac set appkey 00112233445566770011223344556677
loramac saveve
2021-07-07 17:52:08,930 # loramac save
rebootot
2021-07-07 17:52:13,766 # reboot
2021-07-07 17:52:13,853 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 17:52:13,856 # All up, running the shell now
loramac get deveuiui
2021-07-07 17:52:27,518 # loramac get deveui
2021-07-07 17:52:27,520 # DEVEUI: 0011223344556677
loramac get appeuiui
2021-07-07 17:52:31,522 # loramac get appeui
2021-07-07 17:52:31,525 # APPEUI: 0011223344556677
loramac get appkeyey
2021-07-07 17:52:36,473 # loramac get appkey
2021-07-07 17:52:36,477 # APPKEY: 00112233445566770011223344556677
loramac erasese
2021-07-07 17:52:40,474 # loramac erase
loramac set devaddr 0011223333
2021-07-07 17:52:44,944 # loramac set devaddr 00112233
loramac set appskey 0011223344556677001122334455667777
2021-07-07 17:52:55,091 # loramac set appskey 00112233445566770011223344556677
loramac set nwkskey 0011223344556677001122334455667777
2021-07-07 17:52:59,942 # loramac set nwkskey 00112233445566770011223344556677
loramac saveve
2021-07-07 17:53:04,187 # loramac save
> reboot
2021-07-07 17:53:12,834 # reboot
2021-07-07 17:53:12,922 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 17:53:12,924 # All up, running the shell now
loramac get devaddrdr
2021-07-07 17:53:18,351 # loramac get devaddr
2021-07-07 17:53:18,353 # DEVADDR: 00112233
loramac get appskeyey
2021-07-07 17:53:22,470 # loramac get appskey
2021-07-07 17:53:22,474 # APPSKEY: 00112233445566770011223344556677
loramac get nwkskeyey
2021-07-07 17:53:29,927 # loramac get nwkskey
2021-07-07 17:53:29,931 # NWKSKEY: 00112233445566770011223344556677
```

</details>

**Comment by jia200x** [comment]
### [Task #05 - GNRC LoRaWAN (Over The Air Activation)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-05---gnrc-lorawan-over-the-air-activation)

<details><summary><strong>PASSED</strong></summary>

```
ifconfig
2021-07-07 17:56:36,244 # ifconfig
2021-07-07 17:56:36,254 # Iface  3  HWaddr: 00:00:00:00  Frequency: 868299987Hz  RSSI: 99  BW: 125kHz  SF: 7  CR: 4/5  Link: down 
2021-07-07 17:56:36,261 #            TX-Power: 14dBm  State: SLEEP  Demod margin.: 0  Num gateways.: 0 
2021-07-07 17:56:36,263 #           OTAA  
2021-07-07 17:56:36,266 #           L2-PDU:767  
2021-07-07 17:56:36,266 #           
ifconfig 3 set deveui 488038d49a5a156b
2021-07-07 17:56:58,997 # ifconfig 3 set deveui 488038d49a5a156b
2021-07-07 17:56:59,002 # success: set long address on interface 3 to 488038d49a5a156b
ifconfig 3 set appkey 8849748cafbc089b815f70b20b08af02
2021-07-07 17:57:14,730 # ifconfig 3 set appkey 8849748cafbc089b815f70b20b08af02
2021-07-07 17:57:14,737 # success: set AppKey on interface 3 to 8849748cafbc089b815f70b20b08af02
> ifconfig otaa
2021-07-07 17:57:21,355 # ifconfig otaa
2021-07-07 17:57:21,358 # error: invalid interface given
> ifconfig 3 otaa
2021-07-07 17:57:23,672 # ifconfig 3 otaa
2021-07-07 17:57:23,674 # success: set option
> ifconfig 3 up
2021-07-07 17:57:40,517 # ifconfig 3 up
> ifconfig
2021-07-07 17:57:55,711 # ifconfig
2021-07-07 17:57:55,721 # Iface  3  HWaddr: 00:AF:35:84  Frequency: 868099975Hz  RSSI: 99  BW: 125kHz  SF: 12  CR: 4/5  Link: up 
2021-07-07 17:57:55,729 #            TX-Power: 14dBm  State: SLEEP  Demod margin.: 0  Num gateways.: 0 
2021-07-07 17:57:55,731 #           IQ_INVERT  
2021-07-07 17:57:55,734 #           RX_SINGLE  OTAA  L2-PDU:767  
2021-07-07 17:57:55,735 #           
> txtsnd 3 01 test
2021-07-07 17:58:09,003 # txtsnd 3 01 test
> txtsnd 3 01 test2
2021-07-07 17:58:43,629 # txtsnd 3 01 test2
> 2021-07-07 17:58:47,302 # PKTDUMP: data received:
2021-07-07 17:58:47,307 # ~~ SNIP  0 - size:   4 byte, type: NETTYPE_UNDEF (0)
2021-07-07 17:58:47,309 # 00000000  54  45  53  54
2021-07-07 17:58:47,313 # ~~ SNIP  1 - size:   9 byte, type: NETTYPE_NETIF (-1)
2021-07-07 17:58:47,316 # if_pid: 3  rssi: -32768  lqi: 0
2021-07-07 17:58:47,317 # flags: 0x0
2021-07-07 17:58:47,319 # src_l2addr: (nil)
2021-07-07 17:58:47,320 # dst_l2addr: 02
2021-07-07 17:58:47,324 # ~~ PKT    -  2 snips, total size:  13 byte
ifconfig 3 -ack_req
2021-07-07 17:59:24,974 # ifconfig 3 -ack_req
2021-07-07 17:59:24,976 # success: unset option
txtsnd 3 01 test2eq
2021-07-07 17:59:35,631 # txtsnd 3 01 test2
> txtsnd 3 01 test2
2021-07-07 17:59:51,269 # txtsnd 3 01 test2
> txtsnd 3 01 test2
2021-07-07 18:00:03,068 # txtsnd 3 01 test2
> txtsnd 3 01 test2
2021-07-07 18:00:17,337 # txtsnd 3 01 test2
> 2021-07-07 18:00:21,010 # PKTDUMP: data received:
2021-07-07 18:00:21,015 # ~~ SNIP  0 - size:   4 byte, type: NETTYPE_UNDEF (0)
2021-07-07 18:00:21,016 # 00000000  54  45  53  54
2021-07-07 18:00:21,021 # ~~ SNIP  1 - size:   9 byte, type: NETTYPE_NETIF (-1)
2021-07-07 18:00:21,024 # if_pid: 3  rssi: -32768  lqi: 0
2021-07-07 18:00:21,025 # flags: 0x0
2021-07-07 18:00:21,028 # src_l2addr: (nil)
2021-07-07 18:00:21,029 # dst_l2addr: 02
2021-07-07 18:00:21,032 # ~~ PKT    -  2 snips, total size:  13 byte
ifconfig 3 ack_req
2021-07-07 18:00:32,135 # ifconfig 3 ack_req
2021-07-07 18:00:32,137 # success: set option
txtsnd 3 01 test2eq
2021-07-07 18:00:44,851 # txtsnd 3 01 test2
> 2021-07-07 18:00:48,524 # PKTDUMP: data received:
2021-07-07 18:00:48,528 # ~~ SNIP  0 - size:   4 byte, type: NETTYPE_UNDEF (0)
2021-07-07 18:00:48,531 # 00000000  54  45  53  54
2021-07-07 18:00:48,535 # ~~ SNIP  1 - size:   9 byte, type: NETTYPE_NETIF (-1)
2021-07-07 18:00:48,538 # if_pid: 3  rssi: -32768  lqi: 0
2021-07-07 18:00:48,539 # flags: 0x0
2021-07-07 18:00:48,541 # src_l2addr: (nil)
2021-07-07 18:00:48,542 # dst_l2addr: 02
2021-07-07 18:00:48,546 # ~~ PKT    -  2 snips, total size:  13 byte
```

</details>

**Comment by jia200x** [comment]
### [Task #06 - GNRC LoRaWAN (ABP)](https://github.com/RIOT-OS/Release-Specs/blob/394705f179d651f44d133d21832a982df3ed1dad/11-lorawan/README.md#task-06---gnrc-lorawan-abp)

<details><summary><strong>PASSED</strong></summary>


```
2021-07-07 18:03:20,031 # NETOPT_RX_END_IRQ not implemented by driver
2021-07-07 18:03:20,036 # main(): This is RIOT! (Version: 2021.10-devel-HEAD)
2021-07-07 18:03:20,041 # Initialization successful - starting the shell now
ifconfig 3 set addr 008f963a
2021-07-07 18:03:21,194 # ifconfig 3 set addr 008f963a
2021-07-07 18:03:21,199 # success: set (short) address on interface 3 to 008f963a
ifconfig 3 set nwkskey 3cd35f918e7f1232ee0941778ee0238a
2021-07-07 18:03:30,850 # ifconfig 3 set nwkskey 3cd35f918e7f1232ee0941778ee0238a
2021-07-07 18:03:30,857 # success: set NwkSKey on interface 3 to 3cd35f918e7f1232ee0941778ee0238a
ifconfig 3 set appskey 5c140708b42dd430df7adfed1b401cefef
2021-07-07 18:03:36,756 # ifconfig 3 set appskey 5c140708b42dd430df7adfed1b401cef
2021-07-07 18:03:36,763 # success: set AppSKey on interface 3 to 5c140708b42dd430df7adfed1b401cef
> ifconfig 3 up
2021-07-07 18:03:40,683 # ifconfig 3 up
> ifconfig
2021-07-07 18:03:42,847 # ifconfig
2021-07-07 18:03:42,857 # Iface  3  HWaddr: 00:8F:96:3A  Frequency: 868099975Hz  RSSI: 99  BW: 125kHz  SF: 12  CR: 4/5  Link: down 
2021-07-07 18:03:42,864 #            TX-Power: 14dBm  State: TX  Demod margin.: 0  Num gateways.: 0 
2021-07-07 18:03:42,866 #           OTAA  
2021-07-07 18:03:42,869 #           L2-PDU:767  
2021-07-07 18:03:42,870 #           
> ifconfig 3 -otaa
2021-07-07 18:03:57,457 # ifconfig 3 -otaa
2021-07-07 18:03:57,459 # success: unset option
> ifconfig 3 up
2021-07-07 18:03:59,213 # ifconfig 3 up
> ifconfig
2021-07-07 18:04:00,448 # ifconfig
2021-07-07 18:04:00,458 # Iface  3  HWaddr: 00:8F:96:3A  Frequency: 869524963Hz  RSSI: 99  BW: 125kHz  SF: 12  CR: 4/5  Link: up 
2021-07-07 18:04:00,465 #            TX-Power: 14dBm  State: SLEEP  Demod margin.: 0  Num gateways.: 0 
2021-07-07 18:04:00,468 #           IQ_INVERT  
2021-07-07 18:04:00,471 #           RX_SINGLE  L2-PDU:767  
2021-07-07 18:04:00,472 #           
> txtsnd 3 01 test
2021-07-07 18:04:18,933 # txtsnd 3 01 test
ifconfig 3 -ack_req
2021-07-07 18:04:53,990 # ifconfig 3 -ack_req
2021-07-07 18:04:53,993 # success: unset option
txtsnd 3 01 testeq
2021-07-07 18:04:55,505 # txtsnd 3 01 test
> txtsnd 3 01 test
2021-07-07 18:05:21,083 # txtsnd 3 01 test
> txtsnd 3 01 test
2021-07-07 18:05:47,264 # txtsnd 3 01 test
> 2021-07-07 18:05:50,937 # PKTDUMP: data received:
2021-07-07 18:05:50,942 # ~~ SNIP  0 - size:   4 byte, type: NETTYPE_UNDEF (0)
2021-07-07 18:05:50,944 # 00000000  04  F0  B7  7F
2021-07-07 18:05:50,948 # ~~ SNIP  1 - size:   9 byte, type: NETTYPE_NETIF (-1)
2021-07-07 18:05:50,952 # if_pid: 3  rssi: -32768  lqi: 0
2021-07-07 18:05:50,953 # flags: 0x0
2021-07-07 18:05:50,954 # src_l2addr: (nil)
2021-07-07 18:05:50,956 # dst_l2addr: 02
2021-07-07 18:05:50,959 # ~~ PKT    -  2 snips, total size:  13 byte
ifconfig 3 ack_req
2021-07-07 18:06:00,146 # ifconfig 3 ack_req
2021-07-07 18:06:00,148 # success: set option
txtsnd 3 01 test_conf
2021-07-07 18:06:15,377 # txtsnd 3 01 test_conf
> 2021-07-07 18:06:19,214 # PKTDUMP: data received:
2021-07-07 18:06:19,219 # ~~ SNIP  0 - size:   4 byte, type: NETTYPE_UNDEF (0)
2021-07-07 18:06:19,221 # 00000000  25  26  F6  20
2021-07-07 18:06:19,226 # ~~ SNIP  1 - size:   9 byte, type: NETTYPE_NETIF (-1)
2021-07-07 18:06:19,229 # if_pid: 3  rssi: -32768  lqi: 0
2021-07-07 18:06:19,229 # flags: 0x0
2021-07-07 18:06:19,232 # src_l2addr: (nil)
2021-07-07 18:06:19,233 # dst_l2addr: 02
2021-07-07 18:06:19,236 # ~~ PKT    -  2 snips, total size:  13 byte
```

</details>

**Comment by jia200x** [comment]
All LoRaWAN test passed!
We will still need to run them in RC2 because of https://github.com/RIOT-OS/RIOT/pull/16622

**Comment by miri64** [comment]
<h1><a href="https://github.com/RIOT-OS/RIOT/actions/runs/1000935543">Test Report</a></h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by miri64** [comment]
<h1><a href="https://github.com/RIOT-OS/RIOT/actions/runs/1000935543">Test Report</a></h1>


<table>
  <thead>
    <tr><th></th><th>Task</th><th>Outcome</th></tr>
  </thead>
  <tbody>
  </tbody>
</table>

**Comment by MrKevinWeiss** [comment]
Closing in favor of #230
